save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Delete.kt

```diff
@@ -23,6 +23,27 @@ import kotlinx.browser.window
 val deleteSettingsCard: FC<SettingsProps> = FC { props ->
     val deleteUserWindowOpenness = useWindowOpenness()
 
+    val deleteUser = useDeferredRequest {
+        val response = get(
+            url = "$apiUrl/users/delete/${props.userInfo?.name!!}",
+            headers = jsonHeaders,
+            loadingHandler = ::loadingHandler,
+            responseHandler = ::noopResponseHandler,
```

Comment:
To my mind, a response handler is needed here

Reply:
Is there a response handler here?